### ISSUE TITLE
Hash missmatch while installing Processing 3.5.3

### ISSUE THREAD

**Issue by voidIess** [NixOS/nixpkgs#95719]
**Describe the bug**
trying to Install leads to a checksum-missmatch

```
unpacking source archive /build/processing-0269-3.5.3.tar.gz
100 4301k  100 4301k    0     0  34386      0  0:02:08  0:02:08 --:--:-- 27650
hash mismatch in fixed-output derivation '/nix/store/isph8bhw2x5ssznaa0m19vxmykxkzdmk-reference.zip':
  wanted: sha256:198bpk8mzns6w5h0zdf50wr6iv7sgdi6v7jznj5rbsnpgyilxz35
  got:    sha256:19gj6gvszssg3czac5fcn9yg06739k0i9idc4k37k7wp5yqzs510
cannot build derivation '/nix/store/06dhqh4h4887s7wgbyxnamdsnrclzjiq-processing-3.5.3.drv': 1 dependencies couldn't be built
```

**To Reproduce**
NIXPKGS_ALLOW_UNFREE=1 nix-env -iA nixos.processing

**Expected behavior**
Processing should be installed in the user environment

**Notify maintainers**
@tasmo @lopsided98 

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.7.10, NixOS, 20.03.2668.7c85b915892 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.6`
 - channels(root): `"nixos-20.03.2806.cb1996818ed"`
 - channels(voidless): `"unstable-19.09pre188690.8746c77a383"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by Pacman99** [comment]
It looks like this problem was fixed here:  #91398, so it should be fixed for 20.09. Maybe the change could be backported to 20.03, but I'm not sure.
It will definitely work in unstable, if you are able to use that.

**Comment by voidIess** [comment]
Ah allright. On unstable this error seems to be gone.
